presto-docs/src/main/sphinx/release/release-0.223.rst

```diff
@@ -0,0 +1,26 @@
+=============
+Release 0.223
+=============
+
+General Changes
+---------------
+
+* Fix a bug where a connector may not optimize query plan even when an optimizer is specified.
+* Support multibyte characters in ``date_format`` function.
+* Add support for using long polling for task info updates to reduce cpu load on the coordinator.
```

Comment:
Lets expand `info` to say `information`.

Reply:
I think it should be framed in terms of user impact.  Like `Improve coordinator CPU utilization`